src/com/TTT/IPlayer.java

```diff
@@ -0,0 +1,4 @@
+package com.TTT;
+
+public interface IPlayer {
```

Comment:
Historically, interfaces were named `IName` in Java, however these days, with increased readability, it's preferred the `I` is dropped, leaving the interface name `Player`.